data/plugins/findfile.lua

```diff
@@ -19,10 +19,14 @@ command.add(nil, {
       local start, total = system.get_time(), 0
       for i, project in ipairs(core.projects) do
         for project, item in project:files() do
-          if complete or #files > config.plugins.findfile.file_limit then return end
+          if complete or #files > config.plugins.findfile.file_limit then 
```

Comment:
When `complete` is true, we don't possess the `CommandView` anymore, so we shouldn't update.

Reply:
Can't believe you've done this.